### PR TITLE
WIP CentrifugeJS: Updates after runtime refactoring

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Brings CentJS up-to-date with the renamings and refactorings done in the runtime. 

- [x] Update CentJS
- [x] Update App

### Approvals

- [x] Dev

### Impact

CentrifugeJS, Centrifuge App

### CODE REVIEW

File: centrifuge-js/src/modules/pools.ts [hunk 1]
```diff
@@ -1067,17 +1132,17 @@ export function getPoolsModule(inst: CentrifugeBase) {
...
         const loan: Loan = {
           id: loanId,
           poolId,
-          financedAmount: parseBN(loanValue.borrowedAmount),
-          financingFee: parseBN(loanValue.ratePerSec),
-          outstandingDebt: new BN(parseBN(loanValue.principalDebt))
-            .mul(new BN(parseBN(loanValue.accumulatedRate)))
-            .div(new BN(10).pow(new BN(27)))
-            .toString(),
+          financingFee: new Rate(hexToBN(loanValue.interestRatePerSec)),
```
Comment (offerijns) [inline centrifuge-js/src/modules/pools.ts:1140]: We should rename this to:

```suggestion
          interestRatePerSec: new Rate(hexToBN(loanValue.interestRatePerSec)),
```

**Review by offerijns — APPROVED** [review]

File: centrifuge-js/src/modules/pools.ts [hunk 2]
```diff
@@ -1200,24 +1264,28 @@ export function getPoolsModule(inst: CentrifugeBase) {
...
   return hexToBN(value).toString()
 }
 
+function getCurrency(data?: CurrencyData | string) {
+  if (!data) return ''
+  if (typeof data === 'string') return data
+  return Object.keys('permissioned' in data ? data.permissioned! : data)[0]
+}
+
 const tokenNames = [
```
Comment (offerijns) [inline centrifuge-js/src/modules/pools.ts:1281]: This should be removed btw. I added this temporarily for the `simulation-tests`, these token names should be retrieved from the pool config going forward.

**Review by offerijns — APPROVED** [review]

File: centrifuge-js/src/modules/pools.ts (hunk 2 above)
Comment (onnovisser) [reply centrifuge-js/src/modules/pools.ts:1281]: 👍

**Review by sophialittlejohn — APPROVED** [review]
Nice job!! 🎉

**Pull request merged**